html/rendering/non-replaced-elements/form-controls/button-style.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<title>default style on buttons</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
```

Comment:
You should add a comment, using the 'meta' tag and  'assert' id,  to explain briefly what the test tries to verify.

Reply:
Isn't it clear from the title and the test names?